infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuInterClusterReplicationServer.java

```diff
@@ -193,6 +193,8 @@ public class CorfuInterClusterReplicationServer implements Runnable {
                     + "              Max size of replication data message in bytes. \n   "
                     + " --max-write-size=<max-write-size>"
                     + "              Max size of a write in a single corfu transaction.  Integer.MAX_VALUE by default\n "
+                    + "--corfu-port-for-lr=<corfu-port-for-lr>                                   "
+                    + "              Port on which Log Replicator can connect to Corfu Server\n  "
```

Comment:
Add default value, you should be able to get it from here I believe, check: http://docopt.org 

For example:

--speed=<kn>  Speed in knots [default: 10].

Reply:
done